### PR TITLE
Release/mo 1 0 14 5

### PR DESCRIPTION
**Pull request by udaydenduluri33**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: infrastructure/main.tf [hunk 1]
```diff
@@ -5,11 +5,119 @@ locals {
     shared_vault_name = "${var.shared_product_name}-${local.local_env}"
 }
 
+module "app" {
```
Comment (paddybasi) [inline infrastructure/main.tf:8]: Why is this being added back in?

File: src/web.config [hunk 2]
```diff
@@ -9,7 +9,7 @@
...
             <add name="Pragma" value="no-cache" />
             <add name="Expires" value="0" />
-            <add name="Content-Security-Policy" value="default-src 'self'; font-src 'self' https://fonts.gstatic.com data:; img-src 'self' data: https://www.google-analytics.com https://raw.githubusercontent.com/hmcts; style-src https://fonts.googleapis.com https://fonts.gstatic.com 'unsafe-inline' 'self'; script-src 'self' 'unsafe-inline' 'unsafe-eval' www.google-analytics.com www.googletagmanager.com az416426.vo.msecnd.net; connect-src 'self' *.gov.uk dc.services.visualstudio.com; object-src 'self'; frame-ancestors 'none'" />
+            <add name="Content-Security-Policy" value="default-src 'self'; font-src 'self' https://fonts.gstatic.com data:; img-src 'self' data: https://www.google-analytics.com https://raw.githubusercontent.com/hmcts; style-src https://fonts.googleapis.com https://fonts.gstatic.com 'unsafe-inline' 'self'; script-src 'self' 'unsafe-inline' 'unsafe-eval' www.google-analytics.com www.googletagmanager.com az416426.vo.msecnd.net; connect-src 'self' *.gov.uk dc.services.visualstudio.com *.launchdarkly.com; object-src 'self'; frame-ancestors 'none'" />
```
Comment (paddybasi) [inline src/web.config:12]: We really don't need anything going into the web.config file anymore, as we don't use IIS/IISNode

Comment (paddybasi) [reply src/web.config:12]: If anything, this file needs removing

File: src/users/containers/utils/user-roles-util.ts [hunk 3]
```diff
@@ -18,23 +18,25 @@ export class UserRolesUtil {
...
-              if (permission === 'pui-case-manager') {
-                const ccdRolesTobeRemoved = UserRolesUtil.GetRemovableRolesForUser(user, AppConstants.CCD_ROLES);
-                ccdRolesTobeRemoved.forEach(newRole => roles.push(newRole));
+        if (user.roles) {
+          user.roles.forEach( (permission) => {
+              if (!permissions.includes(permission) && !AppConstants.CCD_ROLES.includes(permission)) {
+                roles.push({
+                    name: permission
+                });
+                if (permission === 'pui-case-manager') {
```
Comment (paddybasi) [inline src/users/containers/utils/user-roles-util.ts:29]: Not keen on hardcoded strings, surely can be abstracted out into a config/constant etc

File: src/web.config (hunk 2 above)
Comment (udaydenduluri33) [reply src/web.config:12]: @ernestman28  can you confirm on this? I will remove it.

**Pull request merged**